qutebrowser/misc/backendproblem.py

```diff
@@ -287,6 +287,7 @@ def _try_import_backends():
         assert results.webengine_error is not None
 
     return results
+    # pylint: enable=unused-variable
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.